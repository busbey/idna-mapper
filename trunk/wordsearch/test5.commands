rotate row left 4
