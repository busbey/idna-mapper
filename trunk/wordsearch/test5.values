aaaabaa,1
bbbbcbb,1
ccccdcc,1
ddddedd,1
eeeefee,1
ffffgff,1
gggghgg,1
hhhhihh,1
iiiijii,1
jjjjkjj,1
kkkklkk,1
llllmll,1
mmmmnmm,1
nnnnonn,1
oooopoo,1
ppppapp,1
abcdefghijklmnop,1
bcdefghijklmnopa,1
