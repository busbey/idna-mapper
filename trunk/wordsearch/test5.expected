abcdefghijklmnop
abcdefghijklmnop
abcdefghijklmnop
abcdefghijklmnop
bcdefghijklmnopa
abcdefghijklmnop
abcdefghijklmnop
